tests/settings.py

```diff
@@ -28,6 +28,8 @@
     'django.contrib.admin',
     'django_freeradius',
     'rest_framework',
+    'django_filters',
+    'crispy_forms',
```

Comment:
I sent you a few messages in the public chat regarding this, if not required I suggest to remove it because doesn't seeem to bring much value and will slow down the build by increasing the build time